launch/navigation.launch.py

```diff
@@ -5,63 +5,101 @@
 from ament_index_python.packages import get_package_share_directory
 from launch import LaunchDescription
 from launch.actions import DeclareLaunchArgument
-from launch.actions import IncludeLaunchDescription, GroupAction
+from launch.actions import IncludeLaunchDescription, GroupAction, OpaqueFunction
 from launch.launch_description_sources import PythonLaunchDescriptionSource
 from launch.substitutions import LaunchConfiguration, PythonExpression
 from launch_ros.actions import Node, PushRosNamespace
 from launch.conditions import IfCondition
 
-MY_NEO_ROBOT = os.environ.get('MY_ROBOT', "mpo_700")
-MY_NEO_ENVIRONMENT = os.environ.get('MAP_NAME', "neo_workshop")
+# MY_NEO_ROBOT = os.environ.get('MY_ROBOT', "mpo_700")
+# MY_NEO_ENVIRONMENT = os.environ.get('MAP_NAME', "neo_workshop")
 
 def generate_launch_description():
     use_multi_robots = LaunchConfiguration('use_multi_robots', default='False')
     use_amcl = LaunchConfiguration('use_amcl', default='False')
     use_sim_time = LaunchConfiguration('use_sim_time', default='True')
     namespace = LaunchConfiguration('namespace', default='')
     use_namespace = LaunchConfiguration('use_namespace', default='False')
-    map_dir = LaunchConfiguration(
-        'map',
-        default=os.path.join(
-            get_package_share_directory('neo_simulation2'),
-            'maps',
-            MY_NEO_ENVIRONMENT+'.yaml'))
 
-    param_file_name = 'navigation.yaml'
-    param_dir = LaunchConfiguration(
-        'params_file',
-        default=os.path.join(
-            get_package_share_directory('neo_simulation2'),
-            'configs/'+MY_NEO_ROBOT,
-            param_file_name))
+    # Declare launch arguments 'MY_ROBOT' and 'MAP_NAME' with default values and descriptions
+    declare_my_robot_arg = DeclareLaunchArgument(
+        'MY_ROBOT',
+        default_value='mpo_700',
+        description='Robot Types: "mpo_700", "mpo_500", "mp_400", "mp_500"') 
+       
+    declare_map_name_arg = DeclareLaunchArgument(
+        'MAP_NAME',
+        default_value='neo_workshop',
+        description='Map Types: "neo_track1", "neo_track2", "neo_workshop"')
+    
+    # Create launch configuration variables for the robot and map name
+    MY_NEO_ROBOT_ARG = LaunchConfiguration('MY_ROBOT')
```

Comment:
I would change the `MY_NEO_ROBOT_ARGS` variable to small letter, as well as the `MY_ROBOT` argument itself to small letter..